openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,21 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
+        scope=Scope.content,
```

Comment:
it was `Scope.content`before the change (github just misaligned with file_upload_type_raw). Though I still think it should be content. According to the scope doc (https://github.com/edx/XBlock/blob/master/xblock/fields.py#L165) settings only for one specific run of a course. So if I re-run the course with this block, `scope.settings` will rest?